include/openmc/math_functions.h

```diff
@@ -273,5 +273,10 @@ std::complex<double> faddeeva(std::complex<double> z);
 //! \return Derivative of Faddeeva function evaluated at z
 std::complex<double> w_derivative(std::complex<double> z, int order);
 
+// cvmt evaluate function 
+extern "C" double calc_zn_one_d(int n, double rho, double phi);
```

Comment:
`calc_zn_rad` was already implemented.

Reply:
thx. I am also thinking of replacing it with the existed one.